### ISSUE TITLE
Text showing square symbols on Ubuntu 23.04

### ISSUE THREAD

**Issue by Jervx** [stuartlangridge/ColourPicker#90]
Freshly installed ubuntu 23.04 and after installing ColourPicker the text including the headerbar title is showing square symbols.

![image](https://github.com/stuartlangridge/ColourPicker/assets/48374007/3f37bbaa-d413-4de6-86a1-b93fe27f2523)

but other than that, it still did it's job on picking color. 

![image](https://github.com/stuartlangridge/ColourPicker/assets/48374007/28e0e8d2-5ab0-494f-9ac9-d303d9c0ce98)

**Comment by stuartlangridge** [comment]
Just to confirm, is Pick installed from the snap store?

**Comment by Jervx** [comment]
> Just to confirm, is Pick installed from the snap store?

Sorry I forgot to mention, Yes I installed it via snap.


I use ubuntu on xorg instead of wayland

**Comment by Riaan-ZA** [comment]
I'm getting the same error, pick has been installed via snap store

**Comment by stuartlangridge** [comment]
Yup. I don't think there's anything I can do about this; it happens to a few people using the snap. I reported it upstream (six years ago!) but I don't think there's a fix. https://forum.snapcraft.io/t/snapped-app-not-loading-fonts-on-fedora-and-arch/12484

**Comment by stibu81** [comment]
I had the same problem on Ubuntu 24.04. [This post](https://forum.snapcraft.io/t/snapped-app-not-loading-fonts-on-fedora-and-arch/12484/24) solved my problem:
```
sudo rm /var/cache/fontconfig/*
rm ~/.cache/fontconfig/*
```
